introduction.md

```diff
@@ -0,0 +1 @@
+I am Ivan from Hong kong, I am learning to use github
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I like JavaScript most